appveyor.yml

```diff
@@ -0,0 +1,29 @@
+
+# CI on Windows via appveyor
+# This file was based on Olivier Grisel's python-appveyor-demo
+
+environment:
+
+  global:
+    PYTHON: "C:\\conda"
+    MINICONDA_VERSION: "latest"
+    CONDA_DEPENDENCIES: "numpy scipy matplotlib mayavi sphinx nose pillow"
+
+  matrix:
+    - PYTHON_VERSION: "2.7"
+      PYTHON_ARCH: "64"
+
+platform:
+    -x64
+
+install:
+    - "git clone git://github.com/astropy/ci-helpers.git"
+    - "powershell ci-helpers/appveyor/install-miniconda.ps1"
+    - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+    - "activate test"
```

Comment:
Let's go with this for now, but I am not super happy with this implicit dependence. Experience shows that these things evolve, and we have to track their evolution. As a result just to keep a project alive drains resources. Sphinx-gallery is a small project with little resource.